source/extensions/upstreams/http/well_known_names.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
```

Comment:
Do we actually need to reference these anywhere? In general we are trying to move away from the WKN files, so perhaps just inline the strings where they are used if possible?